signify/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "signify"
+version = "0.4.1"
+authors = ["Jan-Erik Rediger <janerik@fnordig.de>"]
+edition = "2021"
+
+keywords = ["cryptography", "security"]
+description = "Command-line implementation of the signify signature scheme"
+
+readme = "../README.md"
+license = "MIT"
+
+homepage = "https://github.com/badboy/signify-rs"
+repository = "https://github.com/badboy/signify-rs"
+
+[dependencies]
+clap = { version = "3.0.0-beta.5", default-features = false, features = ["cargo", "derive", "std"] }
+rand_core = { version = "0.5", features = ["getrandom"] }
+rpassword = "5"
+libsignify = { path = "../libsignify" } 
```

Comment:
this will require a version before publishing

Reply:
Indeed, yep. I still plan on doing another PR or two to clean and improve things more before going for a release. I think in the last one of those I'll switch this to a version.